backend/src/main/java/com/dinamitaexplosivainsana/nojira/infrastructure/repositories/TaskRepositoryImpl.java

```diff
@@ -17,12 +20,39 @@ public TaskRepositoryImpl(JPATaskRepository taskRepository) {
 
     @Override
     public Task getTaskByTaskId(String taskId) {
-        return null;
+        TaskSchema taskSchema = this.taskRepository.findById(taskId).orElse(null);
+        if (Objects.isNull(taskSchema)) {
+            return null;
+        }
+        return new Task(
+                taskSchema.getId(),
+                taskSchema.getDescription(),
+                taskSchema.getTimeEstimatedInMinutes(),
+                taskSchema.getTitle(),
+                taskSchema.getTimeUsedInMinutes()
+        );
     }
 
     @Override
     public List<Task> getAllTasksByUserId(String userId) {
-        return Collections.emptyList();
+
+        List<TaskSchema> taskSchema = this.taskRepository.getAllTaskByUserId(userId);
+        if (Objects.isNull(taskSchema)){
+            return null;
+        }
+        List<Task> taskList = new ArrayList<>();
+
+        for (TaskSchema taskChange : taskSchema) {
```

Comment:
Si pudieras cambiar este for por un stream sobre taskSchema (que es un mal nombre por cierto) para hacerlo más legible y eficiente estaría genial!!!